.gitignore

```diff
@@ -10,3 +10,4 @@ approval_frame/local_settings.py
 env
 Thumbs.db
 /staticfiles/
+approval_frame/mailchimp_api_key.py
```

Comment:
It would be nice if you could keep the mailchimp_api_key module in the repository and have it read the mailchimp API key through a settings variable. Removing this module currently gives an ImportError while running the project, so that might be an issue when others try to run the project locally.